pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -241,3 +258,88 @@ def cmd(self):
 
         cmd = (" ").join(cmd)
         return cmd
+
+
+class A11yCrawler(BokChoyTestSuite):
+    """
+    Sets up test environment with mega-course loaded, and runs pa11ycralwer
+    against it.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(A11yCrawler, self).__init__(*args, **kwargs)
+
+        self.pa11y_report_dir = os.path.join(self.report_dir, 'pa11ycrawler_reports')
+        self.imports_dir = path('test_root/courses/')
+        self.tar_gz_file = "https://github.com/edx/demo-test-course/archive/master.tar.gz"
+
+    def __enter__(self):
+        self.get_test_course()
+        super(A11yCrawler, self).__enter__()
+
+    def get_test_course(self):
+        """
+        Fetches the test course.
+        """
+        self.imports_dir.makedirs_p()
+        zipped_course = self.imports_dir + 'demo_course.tar.gz'
+
+        msg = colorize('green', "Fetching the test course from github...")
+        print msg
+
+        sh(
+            'wget {tar_gz_file} -O {zipped_course}'.format(
+                tar_gz_file=self.tar_gz_file,
+                zipped_course=zipped_course,
+            )
+        )
```

Comment:
I missed that this was already part of paver. It still might be nice to short circuit the download if the md5s match, but I don't think that's a requirement..